src/diffusers/__init__.py

```diff
@@ -362,26 +369,38 @@
     )
 
 try:
-    if not (is_torch_available() and is_transformers_available() and is_k_diffusion_available()):
+    if not (
```

Comment:
same for all the changes that follow and below

Reply:
Hi. This might be auto formatting plugin induced issue that I am using in VSCode. Will run the command given or manually revert.

Reply:
@a-r-r-o-w I have updated the same manually. I tried use ``make style`` and had some blockers. Please review once and let me know if I manually missed somethings.